src/manifest.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::anyhow;
+use std::path::{Path, PathBuf};
+
+use crate::opts::DEFAULT_MANIFEST_FILE;
+
+pub(crate) fn manifest_path(path: impl AsRef<Path>) -> anyhow::Result<PathBuf> {
```

Comment:
Nit: can we add a doc comment here? It's not immediately clear why this function needs to take a path argument, and I think a document would easily clarify this.

Reply:
Added doc comment and renamed to `resolve_file_path`